apps/pie-microsite/src/pieDesignTokenCategories.json

```diff
@@ -0,0 +1,104 @@
+{
+  "color": {
+    "global": {
+      "whiteBlack": {
+        "displayName": "White and Black"
+      },
+      "orange": {
+        "displayName": "Orange"
+      },
+      "blue": {
+        "displayName": "Blue"
+      },
+      "red": {
+        "displayName": "Red"
+      },
+      "yellow": {
+        "displayName": "Yellow"
+      },
+      "green": {
+        "displayName": "Green"
+      },
+      "purple": {
+        "displayName": "Purple"
+      },
+      "mozzarella": {
+        "displayName": "Mozzarella"
+      },
+      "charcoal": {
+        "displayName": "Charcoal"
+      },
+      "system": {
+        "displayName": "System"
+      }
+    },
+    "alias": {
+      "parentCategories": {
+        "elementary": {
+          "displayName": "Elementary",
+          "description": "Basic alias colour tokens for the basic building blocks of our products, including backgrounds, borders and dividers."
+        },
+        "supportiveFeedback": {
+          "displayName": "Supportive Feedback",
+          "description": "Alias colour tokens used to add meaning and brand our products."
+        },
+        "content": {
+          "displayName": "Content",
+          "description": "Alias colour tokens used for text and icons only."
+        },
+        "reactive": {
+          "displayName": "Reactive",
+          "description": "Alias colour tokens used to represent state changes and loading animations in components."
+        }
+      },
+      "pageBackgrounds": {
```

Comment:
Just a thought around structure here. Would it make sense to group these inside the parentCategories?

I think it probably makes the list a bit more scannable then (not sure if it would a massive pain to rewrite some of the category logic though in this format).

So something like:

```json
"alias": {
      "parentCategories": {
        "elementary": {
          "displayName": "Elementary",
          "description": "Basic alias colour tokens for the basic building blocks of our products, including backgrounds, borders and dividers."
          "subCategories": [
              {
                 "pageBackgrounds": {
                     "displayName": "Page Backgrounds"
                 }
              },
              {
                 "containerBackgrounds": {
                    "displayName": "Container Backgrounds"
                 }
              },
              { "etc" }
          ]
        },
```

Reply:
Just looking at the metadata, it may be a nightmare to not have the categories all in the root of the alias list – if so, then obviously don't worry about it. Just thought it was a nice way of grouping them potentially, but don't want to make the logic more complicated in the code. Take a look and see what you think anyway!

Reply:
Do you mind if we leave this change until we add another token type? This way we can see how spacing/box-shadow affects all these files.

Reply:
Yeh, that's fine. Happy to leave it as-it-is for now